pandas/io/formats/printing.py

```diff
@@ -346,15 +344,17 @@ def format_object_summary(
     # adj can optionally handle unicode eastern asian width
     adj = _get_adjustment()
 
-    def _extend_line(s, line, value, display_width, next_line_prefix):
+    def _extend_line(
+        s: str, line: str, value: str, display_width: int, next_line_prefix: str
+    ) -> Tuple[str, str]:
 
         if adj.len(line.rstrip()) + adj.len(value.rstrip()) >= display_width:
             s += line.rstrip()
             line = next_line_prefix
         line += value
         return s, line
 
-    def best_len(values):
+    def best_len(values: List[Any]) -> int:
```

Comment:
`List[Any]` appeases a type checker but is pretty loose. Is there a way to refine the subtype further?

Reply:
I also don't like this solution, but "mypy " complains about it with every other annotations, that I tried.

Do you have any suggestions?

Reply:
`List[str]` maybe

Reply:
> `List[str]` maybe

It worked!

I wonder why it didn't worked for me yesterday, maybe I was annotating other things and reverted without checking it... anyway, thank you :)